### ISSUE TITLE
from js import document issue

### ISSUE THREAD

**Issue by zeki07k** [pyscript/pyscript#699]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### What happened?
it is tried both python 3.8 and 3.10

Why I can't use **from js import document** line?
My python script doesnt accept it . I mean I cant use **document**
Script :

```
from dimension import Dimension
from js import document

class Board(Dimension):
  BLACK_CELL = "blackCell"
  WHITE_CELL = "white_cell"

def __init__(self, width, height):
    super().__init__(width, height)

def create_html_element(self):
    dimension_size = self.height
    table = document.createElement('table')
    for row in range(dimension_size):
        tr = document.createElement('tr')
        for col in range(dimension_size):
            td = document.createElement('td')
            if (row % 2 == col % 2):
                td.classList.add(self.WHITE_CELL)
            else:
                td.classList.add(self.BLACK_CELL)
            tr.appendChild(td)
        table.appendChild(tr)
    document.body.appendChild(table)

```

### What browsers are you seeing the problem on? (if applicable)

Chrome

### Additional Context

Also I tried **pip install js**, 
However, the result of `print(dir(js))` is :
`['Library', 'Resource', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', 'library']
`

**Comment by JeffersGlass** [comment]
Are you running this script locally, or using it within a <py-script> tag or similar?

"JS" isn't a module on PyPI - it's Pyodide's way of representing the Javascript global namespace, and automatically proxying those objects into Python. See [the Pyodide Getting Started](https://pyodide.org/en/stable/usage/quickstart.html#accessing-javascript-scope-from-python) (and the links therein) for more info.

**Comment by zeki07k** [comment]
I will call mainçpy file via that script:
main.py:
```
from chessBoard.board import Board


def main(width, height):
    board = Board(width, height)
    board.create_html_element()


if __name__ == "main":
    main(8, 8)**

```
then I will put into html file like this.
```
<!DOCTYPE html>
<html>
<head>
    <link rel="stylesheet" href="https://pyscript.net/alpha/pyscript.css" />
    <script defer src="https://pyscript.net/alpha/pyscript.js"></script>
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
</head>
<style>
    body {
        background-color: white;
    }
    .blackCell {
        background-color: black;
        width: 30px;
        height: 30px;
    }
    .whiteCell {
        background-color: white;
        width: 30px;
        height: 30px;
    }
    table {
        border: 3px solid blue;
        width: 270px;
    }
</style>
<table id="board">

</table>

<py-script>

    src="./chess/main/main.py"
</py-script>
</body>
</html>
```

**Comment by JeffersGlass** [comment]
The syntax for including a script via src is `<py-script src="./chess/main/main.py"></py-script>`. The `src` is an attribute of the \<py-script\> tag, not its contents. Though maybe that's a copy-paste issue?

And to double check, you're using a [server](https://github.com/pyscript/pyscript/issues/257#issuecomment-1119595062) of some kind to view these files in a browser, yes?

**Comment by zeki07k** [comment]
In here **from js import document** give an error. this is my actual error. for that reason, I cant go on. 
![image](https://user-images.githubusercontent.com/67226258/185449601-da1a7027-1e6b-4838-8b92-dfdd7f6546fe.png)

**Comment by zeki07k** [comment]
> The syntax for including a script via src is `<py-script src="./chess/main/main.py"></py-script>`. The `src` is an attribute of the <py-script> tag, not its contents. Though maybe that's a copy-paste issue?
> 
> And to double check, you're using a [server](https://github.com/pyscript/pyscript/issues/257#issuecomment-1119595062) of some kind to view these files in a browser, yes?

yes!

**Comment by JeffersGlass** [comment]
Can you copy/paste or screenshot the error that appears in your browser window (or the Developer console)? I think that will help clarify what's happening that it "can't go on."

**Comment by zeki07k** [comment]
basically, the error is here: 
`from js import document
ImportError: cannot import name 'document' from 'js' (D:\sdk\chess10\lib\site-packages\js\__init__.py)`

Also, Is it possible to run main func into pyscript like this? 
![image](https://user-images.githubusercontent.com/67226258/185459121-4433de31-777c-4877-8aff-97b0c3bc51cb.png)

**Comment by JeffersGlass** [comment]
The fact that that package is trying to load from the D: drive indicates that this script isn't running in a browser, but rather the file is being opened from disk. Also, PyScript will not have access to your various folders and files (`./chess...`) unless you using \<py-env\> paths... \<py-env\> to load them into the PyScript environment.

-----

Maybe this is more helpful: here's a minimal example and process for using the document selector from js. If you copy and paste this into a `.html` file and double click on it, it should open showing the text "Hello, World" and, once PyScript initializes, will read "Hello, World! PyScript says Hello!"

```html
<head>
    <link rel="stylesheet" href="https://pyscript.net/latest/pyscript.css" />
    <script defer src="https://pyscript.net/latest/pyscript.js"></script>
  </head>
  <body>
    <div id="my_div">Hello, world!</div>
    <py-script>
      from js import document
      document.getElementById("my_div").innerHTML += " PyScript says hello!"
    </py-script>
  </body>
```

To validate that you're actually using a server, move the above python code into a separate file (let's call it helloworld.py) in the same folder and modify your html as follows:

```python
#helloworld.py
from js import document
document.getElementById("my_div").innerHTML += " PyScript says hello!"
````

```html
<!-- yourfile.html -->
<head>
    <link rel="stylesheet" href="https://pyscript.net/latest/pyscript.css" />
    <script defer src="https://pyscript.net/latest/pyscript.js"></script>
    <py-env>
      - paths:
        - ./helloworld.py
    </py-env>
  </head>
  <body>
    <div id="my_div">Hello, world!</div>
    <py-script>
        import helloworld
    </py-script>
  </body>
```
If you see the **PyScript: Access to local files...** error, then you're opening the files locally or running them as a script, rather than running them in the browser.

To have access to your chess files, you'll need to list them in `paths` like helloworld.py was above, as well as make them accessible using a server.

**Comment by zeki07k** [comment]
I see with that way. Now, I am able to call .py file and can open the server on localhost:8000.

But I am confused python script. that is below.
```
`
from chessBoard.board import Board
print("STEP 1")

def main(width=8, height=8):
    board = Board(width, height)
    board.create_html_element()
print("STEP 2")
`
```
As I run that one, regardless of logic. it returns like this and no error
```
`STEP 1

STEP 2`


```
However, on the server returns this error. can't find the file. I cant figure it out what is  the reason for that??
`JsException(PythonError: Traceback (most recent call last): File "/lib/python3.10/site-packages/_pyodide/_base.py", line 429, in eval_code .run(globals, locals) File "/lib/python3.10/site-packages/_pyodide/_base.py", line 300, in run coroutine = eval(self.code, globals, locals) File "", line 1, in File "/home/pyodide/Main.py", line 1, in from chessBoard.board import Board ModuleNotFoundError: No module named 'chessBoard' )`

**Comment by tedpatrick** [comment]
The `js` module is different depending on what version of PyScript and Pyodide are in use.

This should work as expected in the `unstable` branch which will soon be released to `latest` and `2022.09.1` release

js on `unstable` ✅ 
```html
<html>
<head>
    <link rel="stylesheet" href="https://pyscript.net/unstable/pyscript.css" />
    <script defer src="https://pyscript.net/unstable/pyscript.js"></script>
</head>
<body>
    <py-script>
from js import console, document

console.log( "Hello on JS Console" )
document.title = "FOO"
    </py-script>
</body>
</html>
```

On the module loading issue, you need to load the module via `<py-config>` within `unstable` release of PyScript as follows:
```html
<py-config>
paths = [
 "myModule.py"
]
</py-config>
```

As this resolves the original issue, I am going to close the issue.
If you are seeing additional issues, please file a bug. Thanks.
